keyboards/splitkb/elora/rev1/keyboard.json

```diff
@@ -0,0 +1,108 @@
+{
+    "keyboard_name": "Elora rev1",
+    "bootloader": "rp2040",
+    "encoder": {
+        "driver": "quadrature",
+        "enabled": true
+    },
+    "features": {
+        "pointing_device": true,
+        "rgb_matrix": true,
+        "rgblight": false
+    },
+    "matrix_size": {
+        "cols": 8,
+        "rows": 12
+    },
```

Comment:
```suggestion
```

Reply:
This is still needed right, as it doesn't compile without these? As we don't define any matrix pins directly, it doesn't know the size of the matrix.

Reply:
For custom matrix, the matrix row and column count should be defined in `config.h`, as they are not actually part of the `info.json` schema.